girder/api/rest.py

```diff
@@ -473,6 +478,9 @@ def ensureTokenScopes(token, scope):
     :type scope: str or list of str
     """
     tokenModel = ModelImporter.model('token')
+    if tokenModel.hasScope(token, TokenScope.USER_AUTH):
```

Comment:
USER_AUTH tokens are ones used by actual logged in users, as opposed to tokens being issued on behalf of the user to third-party clients. Users are allowed to do anything whilst logged in as themselves, but third party applications can be granted only specific scopes. So yes, it is a super-scope, but is used to distinguish between normal session tokens and OAuth tokens.